include/ts/ts.h

```diff
@@ -30,6 +30,7 @@
 #pragma once
 
 #include <ts/apidefs.h>
+#include <ts/nexthop.h>
```

Comment:
Can you make this "nexthopdefs.h" to be consistent with "apidefs.h"?

Reply:
Sure. The name is a legacy, I think I'd prefer `parentselectdefs.h` or `parentdefs.h`. I'd like to standardize around "parent select" - we have too many names for the same thing X_X

Reply:
"upstream_select.h" - I think "parent" is a poor legacy name. But it's not that important.

Reply:
I changed it to `parentselectdefs.h` before I saw this comment.
I don't have a strong opinion on the name, as long as it's consistent. If we name that file `upstream_select.h`, I'd like to rename the whole plugin `upstream_select` to match

Reply:
Meh. We're going to have another round of changes before 10 ships, so I'm fine with waiting until we're more confident about those changes. We could also put it off until 11 as well.